qdrant-landing/content/documentation/faq/database-optimization.md

```diff
@@ -34,11 +34,30 @@ As a result, the Qdrant process might use more memory than the minimum required
 
 If you want to limit the memory usage of the service, we recommend using [limits in Docker](https://docs.docker.com/config/containers/resource_constraints/#memory) or Kubernetes.
 
-
 ### My requests are very slow or time out. What should I do?
 
 There are several possible reasons for that:
 
 - **Using filters without payload index** -- If you're performing a search with a filter but you don't have a payload index, Qdrant will have to load whole payload data from disk to check the filtering condition. Ensure you have adequately configured [payload indexes](../../concepts/indexing/#payload-index).
 - **Usage of on-disk vector storage with slow disks** -- If you're using on-disk vector storage, ensure you have fast enough disks. We recommend using local SSDs with at least 50k IOPS. Read more about the influence of the disk speed on the search latency in the article about [Memory Consumption](../../../articles/memory-consumption/).
 - **Large limit or non-optimal query parameters** -- A large limit or offset might lead to significant performance degradation. Please pay close attention to the query/collection parameters that significantly diverge from the defaults. They might be the reason for the performance issues.
+
+### How can I optimize optimizer's configuration settings for better accuracy and speed, especially for large-scale collections?
+
+To optimize optimizer's config in Qdrant for better accuracy and speed, consider the following:
+
+- For low memory footprint with high-speed search, utilize vector quantization with disk storage for vectors and in-memory quantized vectors. Configure memmap_threshold and always_ram accordingly.
+- To prioritize high precision with a low memory footprint, enable on-disk vectors and HNSW index. Adjust HNSW parameters for precision while considering disk IOPS.
```

Comment:
does this answer the question? It feels like a copy-paste from https://qdrant.tech/documentation/guides/optimize/, why don't we just give a link to it?